### PR TITLE
First try for adding the logging to the code (WIP)

### PR DESCRIPTION
**Pull request by bazkiaei**
The goal of this PR is to add the logging mechanism to the code. I will appreciate any advice on the mechanism itself, where and when is the best to use messages and the messages themselves. This PR is related to #28 .

### CODE REVIEW

File: mocks/mocks.py [hunk 1]
```diff
@@ -23,7 +25,15 @@
 from mocks.utils import load_yaml_config
 
 
+def create_logger():
```
Comment (bazkiaei) [inline mocks/mocks.py:71]: Although this is a WIP PR, I appreciate any advice on any stage of it.

**Comment by bazkiaei** [comment]
@AstroHuntsman/spiderlings 
Hi guys, could you please have a first look at this PR?

**Comment by wtgee** [comment]
> @AstroHuntsman/spiderlings
> Hi guys, could you please have a first look at this PR?

@bazkiaei will you also do an actual review request up in the top right of the PR? You can just request from the spiderlings and we will all get a message and then any of us could do it.

**Comment by bazkiaei** [comment]
@wtgee 
I need a huge amount of advice for this PR.

**Comment by lspitler** [comment]
@bazkiaei as we discussed, this is a minor addition that you should just spend a little time on. Logging for this type of program isn't so important at this stage, but good to start now in the beginning, so this looks fine to me.

**Review by lspitler — APPROVED** [review]
@bazkiaei as we discussed, this is a minor addition that you should just spend a little time on. Logging for this type of program isn't so important at this stage, but good to start now in the beginning, so this looks fine to me.

**Pull request merged**